modules/exploits/linux/http/panos_auth_rce.rb

```diff
@@ -0,0 +1,164 @@
+##
```

Comment:
`panos_auth_rce` is a bit of a generic name. There could be another authenticated RCE vulnerability in the future that affects panos. I'd suggest either incorporating the CVE into the name or making it more specific to the vulnerability that's being exploited like `panos_op_cmd_exec`.

Reply:
Totally agree, thanks for pointing this out. Changed to `panos_op_cmd_exec`